component-catalog-connectors/artifactory-connector/artifactory_catalog_connector/artifactory-catalog.json

```diff
@@ -0,0 +1,172 @@
+{
+  "$schema": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/metadata/schemas/meta-schema.json",
+  "$id": "https://raw.githubusercontent.com/elyra-ai/examples/master/component-catalog-connectors/artifactory-connector/artifactory_catalog_connector/artifactory-catalog.json",
+  "title": "Artifactory Component Catalog",
+  "name": "artifactory-catalog",
+  "schemaspace": "component-catalogs",
+  "schemaspace_id": "8dc89ca3-4b90-41fd-adb9-9510ad346620",
+  "metadata_class_name": "elyra.pipeline.component_metadata.ComponentCatalogMetadata",
+  "uihints": {
+    "title": "Artifactory Component Catalog",
+    "icon": "",
+    "reference_url": "https://github.com/elyra-ai/examples/tree/master/component-catalog-connectors/artifactory-connector"
```

Comment:
```suggestion
    "reference_url": "https://github.com/elyra-ai/examples/tree/main/component-catalog-connectors/artifactory-connector"
```

Reply:
Resolved in https://github.com/elyra-ai/examples/pull/99/commits/b97b758ec6867f83c646bb560d12cfb80f7ee7bb